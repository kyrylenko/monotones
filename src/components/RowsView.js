import React, { Component } from 'react';
import { Col, Container, Row } from 'react-bootstrap'; //https://github.com/react-bootstrap/react-bootstrap
import { SoundSlider } from '../components/SoundSlider';
import '../App.css';

export class RowsView extends Component {

    render() {
        return (
            <Container>
                <Row>
                    <Col lg={2} md={2} sm={2} className="hidden-xs"></Col>
                    <Col lg={3} md={3} sm={4} xs={6} className="text-center">
                        <SoundSlider mute={this.props.mute} src={require('../assets/icons/white/summer_day.png')} sound={require('../assets/sounds/summer_day.mp3')} enabled={false} title="Summer day" />
                    </Col>
                    <Col lg={2} md={2} sm={2} className="hidden-xs hidden-sm"></Col>
                    <Col lg={3} md={3} sm={4} xs={6} className="text-center">
                        <SoundSlider mute={this.props.mute} src={require('../assets/icons/white/summer_night.png')} sound={require('../assets/sounds/summer_night.mp3')} enabled={false} title="Summer night" />
                    </Col>
                    <Col lg={2} md={2} sm={2} className="hidden-xs"></Col>
                </Row>
                <Row>
                    <Col lg={2} md={2} sm={2} className="hidden-xs"></Col>
                    <Col lg={3} md={3} sm={4} xs={6} className="text-center">
                        <SoundSlider mute={this.props.mute} src={require('../assets/icons/white/thunderstorm.png')} sound={require('../assets/sounds/thunderstorm.mp3')} enabled={false} title="Thunderstorm" />
                    </Col>
                    <Col lg={2} md={2} sm={2} className="hidden-xs hidden-sm"></Col>
                    <Col lg={3} md={3} sm={4} xs={6} className="text-center">
                        <SoundSlider mute={this.props.mute} src={require('../assets/icons/white/wind_chimes.png')} sound={require('../assets/sounds/wind_chimes.mp3')} enabled={false} title="Wind chimes" />
                    </Col>
                    <Col lg={2} md={2} sm={2} className="hidden-xs"></Col>
                </Row>
                <Row>
                    <Col lg={2} md={2} sm={2} className="hidden-xs"></Col>
                    <Col lg={3} md={3} sm={4} xs={6} className="text-center">
                        <SoundSlider mute={this.props.mute} src={require('../assets/icons/white/sailing_yacht.png')} sound={require('../assets/sounds/sailing_yacht.mp3')} enabled={false} title="Sailing yacht" />
                    </Col>
                    <Col lg={2} md={2} sm={2} className="hidden-xs hidden-sm"></Col>
                    <Col lg={3} md={3} sm={4} xs={6} className="text-center">
                        <SoundSlider mute={this.props.mute} src={require('../assets/icons/white/blue_whales.png')} sound={require('../assets/sounds/bleu_whales.mp3')} enabled={false} title="Bleu whales" />
                    </Col>
                    <Col lg={2} md={2} sm={2} className="hidden-xs"></Col>
                </Row>
                <Row>
                    <Col lg={2} md={2} sm={2} className="hidden-xs"></Col>
                    <Col lg={3} md={3} sm={4} xs={6} className="text-center">
                        <SoundSlider mute={this.props.mute} src={require('../assets/icons/white/campfire.png')} sound={require('../assets/sounds/campfire.mp3')} enabled={false} title="Campfire" />
                    </Col>
                    <Col lg={2} md={2} sm={2} className="hidden-xs hidden-sm"></Col>
                    <Col lg={3} md={3} sm={4} xs={6} className="text-center">
                        <SoundSlider mute={this.props.mute} src={require('../assets/icons/white/farm.png')} sound={require('../assets/sounds/farm.mp3')} enabled={false} title="Farm" />
                    </Col>
                    <Col lg={2} md={2} sm={2} className="hidden-xs"></Col>
                </Row>
                <Row>
                    <Col lg={2} md={2} sm={2} className="hidden-xs"></Col>
                    <Col lg={3} md={3} sm={4} xs={6} className="text-center">
                        <SoundSlider mute={this.props.mute} src={require('../assets/icons/white/deep_space.png')} sound={require('../assets/sounds/deep_space.mp3')} enabled={false} title="Deep space" />
                    </Col>
                    <Col lg={2} md={2} sm={2} className="hidden-xs hidden-sm"></Col>
                    <Col lg={3} md={3} sm={4} xs={6} className="text-center">
                        <SoundSlider mute={this.props.mute} src={require('../assets/icons/white/inside_train.png')} sound={require('../assets/sounds/inside_train.mp3')} enabled={false} title="Inside train" />
                    </Col>
                    <Col lg={2} md={2} sm={2} className="hidden-xs"></Col>
                </Row>
                <Row>
                    <Col lg={2} md={2} sm={2} className="hidden-xs"></Col>
                    <Col lg={3} md={3} sm={4} xs={6} className="text-center">
                        <SoundSlider mute={this.props.mute} src={require('../assets/icons/white/river_stream.png')} sound={require('../assets/sounds/river_stream.mp3')} enabled={false} title="River stream" />
                    </Col>
                    <Col lg={2} md={2} sm={2} className="hidden-xs hidden-sm"></Col>
                    <Col lg={3} md={3} sm={4} xs={6} className="text-center">
                        <SoundSlider mute={this.props.mute} src={require('../assets/icons/white/seawaves.png')} sound={require('../assets/sounds/seawaves.mp3')} enabled={false} title="Seawaves" />
                    </Col>
                    <Col lg={2} md={2} sm={2} className="hidden-xs"></Col>
                </Row>
                <Row>
                    <Col lg={2} md={2} sm={2} className="hidden-xs"></Col>
                    <Col lg={3} md={3} sm={4} xs={6} className="text-center">
                        <SoundSlider mute={this.props.mute} src={require('../assets/icons/white/october_rain.png')} sound={require('../assets/sounds/october_rain.mp3')} enabled={false} title="October rain" />
                    </Col>
                    <Col lg={2} md={2} sm={2} className="hidden-xs hidden-sm"></Col>
                    <Col lg={3} md={3} sm={4} xs={6} className="text-center">
                        <SoundSlider mute={this.props.mute} src={require('../assets/icons/white/paris_cafe.png')} sound={require('../assets/sounds/paris_cafe.mp3')} enabled={false} title="Paris cafe" />
                    </Col>
                    <Col lg={2} md={2} sm={2} className="hidden-xs"></Col>
                </Row>
            </Container>
        );
    }
}