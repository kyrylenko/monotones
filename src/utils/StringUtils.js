export class StringUtils {
    idToTitle = (string) => {
        const newStr = `${string[0].toUpperCase()}${string.slice(1)}`
        return newStr.replace(/_/g, " ");
    }
}