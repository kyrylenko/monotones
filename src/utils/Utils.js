import { relax, sleep, focus } from '../constants/soundIds';
import defaultValues from '../constants/defaultValues';

const n = (n) => n > 9 ? '' + n : '0' + n;

export const secToMin = (sec) => {
    const minutes = n(Math.floor((sec % (1000 * 60 * 60)) / 60));
    const seconds = n(Math.floor(sec % 60));

    return { minutes, seconds };
};

const aggregate = (sounds, reduxSounds) => {
    return Object.values(sounds).map(x => {
        const setting = reduxSounds[x];
        const hasSetting = setting !== undefined;

        return {
            id: x,
            isPlay: hasSetting ? setting.isPlay : false,
            volume: hasSetting ? setting.volume : defaultValues.defaultVolume,
            isLoaded: hasSetting ? setting.isLoaded : false,
        };
    })
};

export const aggregateSounds = (reduxSounds = {}) => {
    const relaxSounds = aggregate(relax, reduxSounds);
    const sleepSounds = aggregate(sleep, reduxSounds);
    const focusSounds = aggregate(focus, reduxSounds);

    return { relaxSounds, sleepSounds, focusSounds };
};