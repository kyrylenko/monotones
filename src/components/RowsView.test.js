import React from 'react';
import RowsView from './RowsView';
import renderer from 'react-test-renderer';
import { aggregateSounds } from '../utils/Utils';

describe('RowsView', () => {
    it('snapshot is correct', () => {
        const { relaxSounds } = aggregateSounds();
        const tree = renderer.create(<RowsView
            sounds={relaxSounds}
            playPauseVolume={() => { }}
            isGlobalPlay={true} />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});