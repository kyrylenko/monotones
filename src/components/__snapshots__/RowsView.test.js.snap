// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RowsView snapshot is correct 1`] = `
<div
  className="container"
>
  <div
    className="sounds-row row justify-content-center"
  >
    <div
      className="text-center col-6 col-sm-4 col-md-3 col-lg-3"
    >
      <img
        alt="summer_day"
        className="sound-icon"
        onClick={[Function]}
        src="summer_day.png"
        style={
          Object {
            "opacity": null,
          }
        }
        title="Summer day"
      />
      <div
        className="rc-slider mx-auto"
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onMouseUp={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "visibility": "hidden",
            "width": "80%",
          }
        }
      >
        <div
          className="rc-slider-rail"
          style={
            Object {
              "backgroundColor": "#CBE7F3",
              "height": "6px",
            }
          }
        />
        <div
          className="rc-slider-track"
          style={
            Object {
              "backgroundColor": "#A3D9F0",
              "height": "6px",
              "left": "0%",
              "width": "30%",
            }
          }
        />
        <div
          className="rc-slider-step"
        />
        <div
          aria-disabled={false}
          aria-valuemax={1}
          aria-valuemin={0}
          aria-valuenow={0.3}
          className="rc-slider-handle"
          onBlur={[Function]}
          onKeyDown={[Function]}
          role="slider"
          style={
            Object {
              "height": 24,
              "left": "30%",
              "marginTop": -9,
              "width": 24,
            }
          }
          tabIndex={0}
        />
        <div
          className="rc-slider-mark"
        />
      </div>
    </div>
    <div
      className="d-none d-sm-none d-md-block col-sm-2 col-md-2 col-lg-2"
    />
    <div
      className="text-center col-6 col-sm-4 col-md-3 col-lg-3"
    >
      <img
        alt="kitty_purr"
        className="sound-icon"
        onClick={[Function]}
        src="kitty_purr.png"
        style={
          Object {
            "opacity": null,
          }
        }
        title="Kitty purr"
      />
      <div
        className="rc-slider mx-auto"
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onMouseUp={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "visibility": "hidden",
            "width": "80%",
          }
        }
      >
        <div
          className="rc-slider-rail"
          style={
            Object {
              "backgroundColor": "#CBE7F3",
              "height": "6px",
            }
          }
        />
        <div
          className="rc-slider-track"
          style={
            Object {
              "backgroundColor": "#A3D9F0",
              "height": "6px",
              "left": "0%",
              "width": "30%",
            }
          }
        />
        <div
          className="rc-slider-step"
        />
        <div
          aria-disabled={false}
          aria-valuemax={1}
          aria-valuemin={0}
          aria-valuenow={0.3}
          className="rc-slider-handle"
          onBlur={[Function]}
          onKeyDown={[Function]}
          role="slider"
          style={
            Object {
              "height": 24,
              "left": "30%",
              "marginTop": -9,
              "width": 24,
            }
          }
          tabIndex={0}
        />
        <div
          className="rc-slider-mark"
        />
      </div>
    </div>
  </div>
  <div
    className="sounds-row row justify-content-center"
  >
    <div
      className="text-center col-6 col-sm-4 col-md-3 col-lg-3"
    >
      <img
        alt="campfire"
        className="sound-icon"
        onClick={[Function]}
        src="campfire.png"
        style={
          Object {
            "opacity": null,
          }
        }
        title="Campfire"
      />
      <div
        className="rc-slider mx-auto"
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onMouseUp={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "visibility": "hidden",
            "width": "80%",
          }
        }
      >
        <div
          className="rc-slider-rail"
          style={
            Object {
              "backgroundColor": "#CBE7F3",
              "height": "6px",
            }
          }
        />
        <div
          className="rc-slider-track"
          style={
            Object {
              "backgroundColor": "#A3D9F0",
              "height": "6px",
              "left": "0%",
              "width": "30%",
            }
          }
        />
        <div
          className="rc-slider-step"
        />
        <div
          aria-disabled={false}
          aria-valuemax={1}
          aria-valuemin={0}
          aria-valuenow={0.3}
          className="rc-slider-handle"
          onBlur={[Function]}
          onKeyDown={[Function]}
          role="slider"
          style={
            Object {
              "height": 24,
              "left": "30%",
              "marginTop": -9,
              "width": 24,
            }
          }
          tabIndex={0}
        />
        <div
          className="rc-slider-mark"
        />
      </div>
    </div>
    <div
      className="d-none d-sm-none d-md-block col-sm-2 col-md-2 col-lg-2"
    />
    <div
      className="text-center col-6 col-sm-4 col-md-3 col-lg-3"
    >
      <img
        alt="river_stream"
        className="sound-icon"
        onClick={[Function]}
        src="river_stream.png"
        style={
          Object {
            "opacity": null,
          }
        }
        title="River stream"
      />
      <div
        className="rc-slider mx-auto"
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onMouseUp={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "visibility": "hidden",
            "width": "80%",
          }
        }
      >
        <div
          className="rc-slider-rail"
          style={
            Object {
              "backgroundColor": "#CBE7F3",
              "height": "6px",
            }
          }
        />
        <div
          className="rc-slider-track"
          style={
            Object {
              "backgroundColor": "#A3D9F0",
              "height": "6px",
              "left": "0%",
              "width": "30%",
            }
          }
        />
        <div
          className="rc-slider-step"
        />
        <div
          aria-disabled={false}
          aria-valuemax={1}
          aria-valuemin={0}
          aria-valuenow={0.3}
          className="rc-slider-handle"
          onBlur={[Function]}
          onKeyDown={[Function]}
          role="slider"
          style={
            Object {
              "height": 24,
              "left": "30%",
              "marginTop": -9,
              "width": 24,
            }
          }
          tabIndex={0}
        />
        <div
          className="rc-slider-mark"
        />
      </div>
    </div>
  </div>
  <div
    className="sounds-row row justify-content-center"
  >
    <div
      className="text-center col-6 col-sm-4 col-md-3 col-lg-3"
    >
      <img
        alt="farm"
        className="sound-icon"
        onClick={[Function]}
        src="farm.png"
        style={
          Object {
            "opacity": null,
          }
        }
        title="Farm"
      />
      <div
        className="rc-slider mx-auto"
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onMouseUp={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "visibility": "hidden",
            "width": "80%",
          }
        }
      >
        <div
          className="rc-slider-rail"
          style={
            Object {
              "backgroundColor": "#CBE7F3",
              "height": "6px",
            }
          }
        />
        <div
          className="rc-slider-track"
          style={
            Object {
              "backgroundColor": "#A3D9F0",
              "height": "6px",
              "left": "0%",
              "width": "30%",
            }
          }
        />
        <div
          className="rc-slider-step"
        />
        <div
          aria-disabled={false}
          aria-valuemax={1}
          aria-valuemin={0}
          aria-valuenow={0.3}
          className="rc-slider-handle"
          onBlur={[Function]}
          onKeyDown={[Function]}
          role="slider"
          style={
            Object {
              "height": 24,
              "left": "30%",
              "marginTop": -9,
              "width": 24,
            }
          }
          tabIndex={0}
        />
        <div
          className="rc-slider-mark"
        />
      </div>
    </div>
    <div
      className="d-none d-sm-none d-md-block col-sm-2 col-md-2 col-lg-2"
    />
    <div
      className="text-center col-6 col-sm-4 col-md-3 col-lg-3"
    >
      <img
        alt="blue_whales"
        className="sound-icon"
        onClick={[Function]}
        src="blue_whales.png"
        style={
          Object {
            "opacity": null,
          }
        }
        title="Blue whales"
      />
      <div
        className="rc-slider mx-auto"
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onMouseUp={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "visibility": "hidden",
            "width": "80%",
          }
        }
      >
        <div
          className="rc-slider-rail"
          style={
            Object {
              "backgroundColor": "#CBE7F3",
              "height": "6px",
            }
          }
        />
        <div
          className="rc-slider-track"
          style={
            Object {
              "backgroundColor": "#A3D9F0",
              "height": "6px",
              "left": "0%",
              "width": "30%",
            }
          }
        />
        <div
          className="rc-slider-step"
        />
        <div
          aria-disabled={false}
          aria-valuemax={1}
          aria-valuemin={0}
          aria-valuenow={0.3}
          className="rc-slider-handle"
          onBlur={[Function]}
          onKeyDown={[Function]}
          role="slider"
          style={
            Object {
              "height": 24,
              "left": "30%",
              "marginTop": -9,
              "width": 24,
            }
          }
          tabIndex={0}
        />
        <div
          className="rc-slider-mark"
        />
      </div>
    </div>
  </div>
  <div
    className="sounds-row row justify-content-center"
  >
    <div
      className="text-center col-6 col-sm-4 col-md-3 col-lg-3"
    >
      <img
        alt="magic_chimes"
        className="sound-icon"
        onClick={[Function]}
        src="magic_chimes.png"
        style={
          Object {
            "opacity": null,
          }
        }
        title="Magic chimes"
      />
      <div
        className="rc-slider mx-auto"
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onMouseUp={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "visibility": "hidden",
            "width": "80%",
          }
        }
      >
        <div
          className="rc-slider-rail"
          style={
            Object {
              "backgroundColor": "#CBE7F3",
              "height": "6px",
            }
          }
        />
        <div
          className="rc-slider-track"
          style={
            Object {
              "backgroundColor": "#A3D9F0",
              "height": "6px",
              "left": "0%",
              "width": "30%",
            }
          }
        />
        <div
          className="rc-slider-step"
        />
        <div
          aria-disabled={false}
          aria-valuemax={1}
          aria-valuemin={0}
          aria-valuenow={0.3}
          className="rc-slider-handle"
          onBlur={[Function]}
          onKeyDown={[Function]}
          role="slider"
          style={
            Object {
              "height": 24,
              "left": "30%",
              "marginTop": -9,
              "width": 24,
            }
          }
          tabIndex={0}
        />
        <div
          className="rc-slider-mark"
        />
      </div>
    </div>
    <div
      className="d-none d-sm-none d-md-block col-sm-2 col-md-2 col-lg-2"
    />
    <div
      className="text-center col-6 col-sm-4 col-md-3 col-lg-3"
    >
      <img
        alt="night_owls"
        className="sound-icon"
        onClick={[Function]}
        src="night_owls.png"
        style={
          Object {
            "opacity": null,
          }
        }
        title="Night owls"
      />
      <div
        className="rc-slider mx-auto"
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onMouseUp={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "visibility": "hidden",
            "width": "80%",
          }
        }
      >
        <div
          className="rc-slider-rail"
          style={
            Object {
              "backgroundColor": "#CBE7F3",
              "height": "6px",
            }
          }
        />
        <div
          className="rc-slider-track"
          style={
            Object {
              "backgroundColor": "#A3D9F0",
              "height": "6px",
              "left": "0%",
              "width": "30%",
            }
          }
        />
        <div
          className="rc-slider-step"
        />
        <div
          aria-disabled={false}
          aria-valuemax={1}
          aria-valuemin={0}
          aria-valuenow={0.3}
          className="rc-slider-handle"
          onBlur={[Function]}
          onKeyDown={[Function]}
          role="slider"
          style={
            Object {
              "height": 24,
              "left": "30%",
              "marginTop": -9,
              "width": 24,
            }
          }
          tabIndex={0}
        />
        <div
          className="rc-slider-mark"
        />
      </div>
    </div>
  </div>
  <div
    className="sounds-row row justify-content-center"
  >
    <div
      className="text-center col-6 col-sm-4 col-md-3 col-lg-3"
    >
      <img
        alt="playground"
        className="sound-icon"
        onClick={[Function]}
        src="playground.png"
        style={
          Object {
            "opacity": null,
          }
        }
        title="Playground"
      />
      <div
        className="rc-slider mx-auto"
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onMouseUp={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "visibility": "hidden",
            "width": "80%",
          }
        }
      >
        <div
          className="rc-slider-rail"
          style={
            Object {
              "backgroundColor": "#CBE7F3",
              "height": "6px",
            }
          }
        />
        <div
          className="rc-slider-track"
          style={
            Object {
              "backgroundColor": "#A3D9F0",
              "height": "6px",
              "left": "0%",
              "width": "30%",
            }
          }
        />
        <div
          className="rc-slider-step"
        />
        <div
          aria-disabled={false}
          aria-valuemax={1}
          aria-valuemin={0}
          aria-valuenow={0.3}
          className="rc-slider-handle"
          onBlur={[Function]}
          onKeyDown={[Function]}
          role="slider"
          style={
            Object {
              "height": 24,
              "left": "30%",
              "marginTop": -9,
              "width": 24,
            }
          }
          tabIndex={0}
        />
        <div
          className="rc-slider-mark"
        />
      </div>
    </div>
    <div
      className="d-none d-sm-none d-md-block col-sm-2 col-md-2 col-lg-2"
    />
    <div
      className="text-center col-6 col-sm-4 col-md-3 col-lg-3"
    >
      <img
        alt="ship_hull"
        className="sound-icon"
        onClick={[Function]}
        src="ship_hull.png"
        style={
          Object {
            "opacity": null,
          }
        }
        title="Ship hull"
      />
      <div
        className="rc-slider mx-auto"
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onMouseUp={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "visibility": "hidden",
            "width": "80%",
          }
        }
      >
        <div
          className="rc-slider-rail"
          style={
            Object {
              "backgroundColor": "#CBE7F3",
              "height": "6px",
            }
          }
        />
        <div
          className="rc-slider-track"
          style={
            Object {
              "backgroundColor": "#A3D9F0",
              "height": "6px",
              "left": "0%",
              "width": "30%",
            }
          }
        />
        <div
          className="rc-slider-step"
        />
        <div
          aria-disabled={false}
          aria-valuemax={1}
          aria-valuemin={0}
          aria-valuenow={0.3}
          className="rc-slider-handle"
          onBlur={[Function]}
          onKeyDown={[Function]}
          role="slider"
          style={
            Object {
              "height": 24,
              "left": "30%",
              "marginTop": -9,
              "width": 24,
            }
          }
          tabIndex={0}
        />
        <div
          className="rc-slider-mark"
        />
      </div>
    </div>
  </div>
  <div
    className="sounds-row row justify-content-center"
  >
    <div
      className="text-center col-6 col-sm-4 col-md-3 col-lg-3"
    >
      <img
        alt="subway"
        className="sound-icon"
        onClick={[Function]}
        src="subway.png"
        style={
          Object {
            "opacity": null,
          }
        }
        title="Subway"
      />
      <div
        className="rc-slider mx-auto"
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onMouseUp={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "visibility": "hidden",
            "width": "80%",
          }
        }
      >
        <div
          className="rc-slider-rail"
          style={
            Object {
              "backgroundColor": "#CBE7F3",
              "height": "6px",
            }
          }
        />
        <div
          className="rc-slider-track"
          style={
            Object {
              "backgroundColor": "#A3D9F0",
              "height": "6px",
              "left": "0%",
              "width": "30%",
            }
          }
        />
        <div
          className="rc-slider-step"
        />
        <div
          aria-disabled={false}
          aria-valuemax={1}
          aria-valuemin={0}
          aria-valuenow={0.3}
          className="rc-slider-handle"
          onBlur={[Function]}
          onKeyDown={[Function]}
          role="slider"
          style={
            Object {
              "height": 24,
              "left": "30%",
              "marginTop": -9,
              "width": 24,
            }
          }
          tabIndex={0}
        />
        <div
          className="rc-slider-mark"
        />
      </div>
    </div>
    <div
      className="d-none d-sm-none d-md-block col-sm-2 col-md-2 col-lg-2"
    />
    <div
      className="text-center col-6 col-sm-4 col-md-3 col-lg-3"
    >
      <img
        alt="tropical_birds"
        className="sound-icon"
        onClick={[Function]}
        src="tropical_birds.png"
        style={
          Object {
            "opacity": null,
          }
        }
        title="Tropical birds"
      />
      <div
        className="rc-slider mx-auto"
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onMouseUp={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "visibility": "hidden",
            "width": "80%",
          }
        }
      >
        <div
          className="rc-slider-rail"
          style={
            Object {
              "backgroundColor": "#CBE7F3",
              "height": "6px",
            }
          }
        />
        <div
          className="rc-slider-track"
          style={
            Object {
              "backgroundColor": "#A3D9F0",
              "height": "6px",
              "left": "0%",
              "width": "30%",
            }
          }
        />
        <div
          className="rc-slider-step"
        />
        <div
          aria-disabled={false}
          aria-valuemax={1}
          aria-valuemin={0}
          aria-valuenow={0.3}
          className="rc-slider-handle"
          onBlur={[Function]}
          onKeyDown={[Function]}
          role="slider"
          style={
            Object {
              "height": 24,
              "left": "30%",
              "marginTop": -9,
              "width": 24,
            }
          }
          tabIndex={0}
        />
        <div
          className="rc-slider-mark"
        />
      </div>
    </div>
  </div>
</div>
`;
